specification/eventhub/resource-manager/Microsoft.EventHub/2017-04-01/examples/NameSpaces/EHNameSpaceDelete.json

```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {
     "namespaceName": "sdk-Namespace-5849",
-    "resourceGroupName": "South Central US",
+    "resourceGroupName": "SouthCentralUS",
```

Comment:
bug wasn't picked up before the reorg since the glob pattern didn't match nested sample folders